akvo/rsr/management/commands/needs_funding.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+
+# Akvo Reporting is covered by the GNU Affero General Public License.
+# See more details in the license.txt file located at the root folder of the Akvo RSR module.
+# For additional details on the GNU license please see < http://www.gnu.org/licenses/agpl.html >.
+
+from decimal import Decimal, InvalidOperation
+from django.core.management.base import BaseCommand
+from optparse import make_option
+from ...models import Project
+
+
+class Command(BaseCommand):
+    args = '<value>'
+    help = 'Checks all projects with a funding between 0 and a specified value'
+
+    option_list = BaseCommand.option_list + (
+        make_option(
+            '--zero',
+            action='store_true',
+            dest='zero',
+            default=False,
+            help='Set needs funding to 0, instead of only printing results'),
+        make_option(
+            '--complete',
+            action='store_true',
+            dest='complete',
+            default=False,
+            help='Set projects to the \'Complete\' status'),
+    )
+
+    def __init__(self, *args, **kwargs):
+        self.value = None
+        self.projects = None
+        super(Command, self).__init__(*args, **kwargs)
+
+    def get_projects(self):
+        self.projects = Project.objects.filter(funds_needed__gt=0., funds_needed__lt=self.value)
+
+    def print_projects(self):
+        self.stdout.write('')
+        self.stdout.write('Found {0} project{1}...'.format(
+            str(self.projects.count()),
+            's' if not self.projects.count() == 1 else ''))
+
+        if self.projects.count() > 0:
+            self.stdout.write('')
+            self.stdout.write('id   |funds_needed |')
+            self.stdout.write('-----|-------------|')
+
+            for p in self.projects:
+                nr_pspaces = 5 - len(str(p.pk))
+                nr_fspaces = 12 - len(str(p.funds_needed))
+                self.stdout.write(str(p.pk) + nr_pspaces * ' ' + '|' + nr_fspaces * ' ' +
```

Comment:
Read up on string formatting (the [examples](https://docs.python.org/2/library/string.html#format-examples) say a lot) and then replace lines 52 - 55 with something approximately like:
```python
self.stdout.write(
        '{pk:{fill}{align}6}|{funds:{fill}{align}13}|'.format(pk=p.pk, funds=p.funds_needed, fill=' ', align='>'))
```